src/redux/modules/posts.js

```diff
@@ -146,9 +154,18 @@ const reducer = (state = initialState, action) => {
         loading: false,
         error: action.error,
       };
+    case SELECT_POST:
+      return {
+        ...state,
+        loading: true,
+        error: null,
+        selectedPost: action.id
+          ? Object.assign({}, state.items.find(item => item.id === action.id))
```

Comment:
We're lazy so we prefer `{...state.items.find(state.items.find(item => item.id === action.id))}`.

Also why do you create a new object here? Do you want to prevent `selectedPost` from being `null` if there is no `item` with the given `id`?